frontend/src/bundles/auth/store/slice.ts

```diff
@@ -5,7 +5,7 @@ import { type ValueOf } from '~/bundles/common/types/types.js';
 
 import { signUp } from './actions.js';
 
-type User = object;
+type User = { email: string; avatarUrl: string };
```

Comment:
I think you should use type AuthResponseDto from **shared** in order to keep it same for whole project
`type AuthResponseDto = {
    user: UserAuthResponseDto;
    token: string;
};`

`type UserAuthResponseDto = {
    id: number;
    email: string;
};`
There is no avatarUrl in UserAuthResponseDto, I think @SofiiaTrokhymchuk @jhladkov backend auth team should add this on the backend

Reply:
Oh, I can add it now. Thank you

Reply:
Thanks) Made as you suggested, hope now will be fine)